sycl/test-e2e/format.py

```diff
@@ -98,6 +98,53 @@ def getMatchedFromList(self, features, alist):
         except ValueError as e:
             raise ValueError("Error in UNSUPPORTED list:\n%s" % str(e))
 
+    def make_default_features_list(self, expr, triple, add_default=True):
+        # Dictionary of features which we know are always/never present for a
+        # given triple (or the system in general).
```

Comment:
The is unclear. I think readability would be much better if you split this map into two sets, one for always, another for never.

That said, I'm not sure if I'd want to see these in one of the earlier patches. IMO, it's better to create some temporary explicit markup to disable tests first, and only after that try to deduce the rules implicitly.

Reply:
Ok, i've split this into two in the latest commit. 

If you have a strong preference for not introducing the logic to mark tests as unsupported just yet we will need to mark the 68 failing tests from this [run](https://github.com/intel/llvm/actions/runs/11388738602) as `REQUIRES: run-mode`. I figured I would this logic to this pr since we will eventually need to introduce something to correctly react to `SUPPORTED` and `REQUIRES`.

Reply:
going to leave out the triple selection logic from this pr